test/WasmSpec/baselines/testsuite/js-api/global/constructor.any.baseline

```diff
@@ -1,38 +1,38 @@
 Harness Status: OK
-Found 36 tests: Fail = 32 Pass = 4
-Fail name  Object expected
-Fail length  Object expected
-Pass No arguments  
+Found 36 tests: Fail = 35 Pass = 1
+Fail name  assert_equals: WebAssembly.Global name should be Global expected "Global" but got ""
+Fail length  assert_equals: WebAssembly.Global length should be 1 expected 1 but got 0
+Fail No arguments  assert_throws: function "() => new WebAssembly.Global()" did not throw

```

Comment:
These tests were incorrectly passing because the test was throwing an exception because WebAssembly.Global doesn't exist